pkg/vmimage/data/script.sh

```diff
@@ -157,6 +158,7 @@ yum -y install \
     rootfiles \
     rsyslog \
     samba-client \
+    screen \
```

Comment:
what do we use `screen` for?

Reply:
This is one of those check boxes.  `screen` allows you to lock your terminal with a password.  It doesn't mean we need it but satisfies one of the `scap` requirements.  We aren't going to be ssh'ing but if we end up needing to debug via ssh it can be very useful.